tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1960,18 +1973,8 @@ tensorflow::Status ConvertReshape(OpConverterParams* params) {
 tensorflow::Status ConvertExpandDims(OpConverterParams* params) {
   const auto& inputs = params->inputs;
   const auto& node_def = params->node_def;
-  if (inputs.size() != 2) {
-    return tensorflow::errors::InvalidArgument(
-        "Two inputs expected for ExpandDims, at ", node_def.name());
-  }
-  if (inputs.at(0).is_weights()) {
-    return tensorflow::errors::Unimplemented(
-        "ExpandDims expects tensor for input, at ", node_def.name());
-  }
-  if (!inputs.at(1).is_weights()) {
-    return tensorflow::errors::InvalidArgument(
-        "ExpandDims expects weights for axis, at ", node_def.name());
-  }
+  TF_RETURN_IF_ERROR(
+      CheckInputsWeights(params, {{"input", false}, {"axis", true}}));
```

Comment:
s/axis/dim?

Reply:
Looks like dim is being deprecated: https://www.tensorflow.org/api_docs/python/tf/expand_dims

Reply:
Ah, did not realize that. I was looking at the REGISTER_OP macro instead.   
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/array_ops.cc#L1932

Given that the conversion takes the TensorFlow graph as input, it makes more sense to use input names from the op. But, using names from the python signature might be more intuitive to users. what do you think?

Reply:
Oh strange, I was originally looking at the C++ API where it is also axis: https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/expand-dims

I have mostly been using the C++ API, but you are right that most users will be using python. Although, most users will not be seeing these messages because they are VLOG(1) - I think its fine either way as long as we are consistent. It may be harder to be consistent with the python signature since there are many python functions with different argument names that call the same  C++ op.

Reply:
Seems it is renamed in api_def as well. That explains it.
https://github.com/tensorflow/tensorflow/blob/0b9429168e73df0fbec61cc9176913920d183ac2/tensorflow/core/api_def/base_api/api_def_ExpandDims.pbtxt#L5

In that case, it should be good to follow either of the name consistently.